requirements.txt

```diff
@@ -4,6 +4,7 @@ decorator==4.3.0
 docutils==0.14
 ebaysdk==2.1.5
 feedparser==5.2.1
+freezegun==0.3.11
```

Comment:
> @d-fence I forgot again, don't we need to update `setup.py`, `debian/control`, `setup/win32/winpy_requirements.txt`, `setup.cfg`, etc. as well? And if we don't, shouldn't we delete those files?

We don't really need to update them because freezegun is only for use in tests, so it's not a packaging requirement.
I will delete `setup/win32/winpy_requirements.txt` in another PR as it's more annoying than useful.

Reply:
> We don't really need to update them because freezegun is only for use in tests, so it's not a packaging requirement.

Hmm, I thought it might be nice to include it in the packages, as the dependency is not too exotic. Just in case the user wants to try `--test-enable`, or installs one of the test modules, for example. Do you think it is a problem to ship in all builds, just like other hard dependencies?